testsuite/MDAnalysisTests/analysis/test_psa.py

```diff
@@ -171,17 +171,13 @@ class TestPSAExceptions(TestCase):
     def test_get_path_metric_func_bad_key(self):
         '''Test that KeyError is caught by
         get_path_metric_func().'''
-
-        try:
+        with self.assertRaises(KeyError):
```

Comment:
Yeah, I put my changes on hold since, once I rebased properly, realized everything is now pytest-specific. Hoping to finish this PR within the week.